modules/core/src/main/java/org/mycontroller/standalone/gateway/model/GatewayPhilipsHue.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2015-2016 Jeeva Kandasamy (jkandasa@gmail.com)
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mycontroller.standalone.gateway.model;
+
+import java.util.HashMap;
+
+import org.mycontroller.standalone.db.tables.GatewayTable;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+/**
+ * @author Fraid(https://github.com/Fraid)
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true, exclude = { "privateKey" })
```

Comment:
@Fraid There is no `privateKey` field in this class. We can remove `exclude = { "privateKey" }`.